CI: bundle install in step cucumber fails

See https://github.com/vcr/vcr/pull/994 for more details

Not much to go on. The Ruby 2.7 complains of "circular deps". 


<details>

```
Fetching gem metadata from https://rubygems.org/...........
Resolving dependencies...
Using rake 13.0.6
Using base64 0.1.1
Using ruby2_keywords 0.0.5
Fetching bigdecimal 3.1.4
Fetching concurrent-ruby 1.2.2
Fetching connection_pool 2.4.1
Fetching minitest 5.20.0
Installing bigdecimal 3.1.4 with native extensions
Installing connection_pool 2.4.1
Using mutex_m 0.1.2
Using public_suffix 5.0.3
Using childprocess 3.0.0
Using contracts 0.17
Using builder 3.2.4
Fetching middleware 0.1.0
Installing minitest 5.20.0
Installing concurrent-ruby 1.2.2
Using thor 1.2.2
Fetching thread_safe 0.3.6
Installing middleware 0.1.0
Fetching cucumber-tag-expressions 2.0.4
Installing thread_safe 0.3.6
Installing cucumber-tag-expressions 2.0.4
Fetching cucumber-cucumber-expressions 10.3.0
Using diff-lcs 1.5.0
Using multi_test 0.1.2
Using ffi 1.16.3
Using rspec-support 3.12.1
Using bundler 2.4.10
Using docile 1.4.0
Using simplecov-html 0.12.3
Using simplecov_json_formatter 0.1.4
Using coderay 1.1.3
Using rexml 3.2.6
Using excon 0.104.0
Using faraday-em_http 1.0.0
Using faraday-em_synchrony 1.0.0
Using faraday-excon 1.1.0
Using faraday-httpclient 1.0.1
Fetching cucumber-html-formatter 6.0.3
Installing cucumber-gherkin 13.0.0
Fetching cucumber-core 7.0.0
Installing cucumber-html-formatter 6.0.3
Installing cucumber-core 7.0.0
Fetching cucumber-wire 3.0.0
Installing cucumber-wire 3.0.0
Fetching cucumber 4.0.0
Installing cucumber 4.0.0
Using aruba 0.14.14
Bundle complete! 22 Gemfile dependencies, 81 gems now installed.
Bundled gems are installed into `./vendor/bundle`
Error: Process completed with exit code 1.
```

</details>

Ruby all of them
Gem current

--- a/Gemfile.cucumber
+++ b/Gemfile.cucumber
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem "cucumber", "~> 4.0"
+gem "cucumber", "~> 9.0"
 gem "webrick"
 
 gemspec

